subscriber.go

```diff
@@ -663,7 +721,7 @@ func (h *handler) handle(ctx context.Context, nextCid cid.Cid, sel ipld.Node, wr
 
 	// Tell the Subscriber to distribute SyncFinished to all notification
 	// destinations.
-	h.subscriber.inEvents <- SyncFinished{Cid: nextCid, PeerID: h.peerID}
+	h.subscriber.inEvents <- SyncFinished{Cid: nextCid, PeerID: h.peerID, SyncedCids: syncedCids}
```

Comment:
Do we need both `Cid` and `SyncedCids`? Or will the `nextCid` always be the first item in `syncedCids`?

Reply:
`nextCid` will always be the first item in syncedCids. I didn't want to change the api, but I think we can do this.

Reply:
We can do this. It's just a bigger breaking change. Maybe in another PR? What do you think?

Reply:
That's fine